/*
  Exercise:

  Let's implement a `boo()` function.
  This function returns a promise that will reject with the string "Boo!"
  using `Promise.reject()`.

  Example:

  ```js
  boo()
    .catch((err) => {
      console.log(`Error is: ${err}`)
    })
  ```

  Should print:

  ```plain
  Error is: Boo!
  ```

  Test your solution with:

  > npm run ex -- 07-promises/exercises/02-reject-promise.test.js
*/

/**
 * @returns {Promise<*>}
 */
export default function boo () {
  // Write your code here!
}
